.github/workflows/deploy.yaml

```diff
@@ -33,8 +33,10 @@ jobs:
         run: |
           if [[ $GITHUB_REF == 'refs/heads/main' ]]; then
             echo DEPLOYMENT_TYPE=prod >> $GITHUB_ENV
-          else
-            echo DEPLOYMENT_TYPE=dev >> $GITHUB_ENV
+          elif [[ $GITHUB_REF == 'refs/heads/deploy' ]]; then
+            echo DEPLOYMENT_TYPE=staging >> $GITHUB_ENV
```

Comment:
Should we rename this to avoid confusion with the `DEPLOYMENT_TYPE` environment variable that seqr uses (see the startup scripts)?

Reply:
Yeah that sounds reasonable, does `DEPLOY_TARGET` seem reasonable?

Reply:
Pushed changes to address this.